Xext/xf86bigfont.c

```diff
@@ -179,7 +179,11 @@ XFree86BigfontExtensionInit()
             + (unsigned int) (65536.0 / (RAND_MAX + 1.0) * rand());
         /* fprintf(stderr, "signature = 0x%08X\n", signature); */
 
+#ifdef XFONT2
+        FontShmdescIndex = xfont2_allocate_font_private_index();
+#else
```

Comment:
since all xfont2 specific functions seem to behave identically, except for the name, i'd add a xfont2_compat.h which looks like

```c
#ifdef XFONT2
define AllocateFontPrivateIndex xfont2_allocate_font_private_index
...
#endif
```
that way  the ifdefs are in a single place and not littered all over the code.
(one could also use function-like macros)